server/checker/replica_checker.go

```diff
@@ -102,7 +106,7 @@ func (r *ReplicaChecker) Check(region *core.RegionInfo) *operator.Operator {
 
 // SelectBestReplacementStore returns a store id that to be used to replace the old peer and distinct score.
 func (r *ReplicaChecker) SelectBestReplacementStore(region *core.RegionInfo, oldPeer *metapb.Peer, filters ...filter.Filter) (uint64, float64) {
-	filters = append(filters, filter.NewExcludedFilter(nil, region.GetStoreIds()))
+	filters = append(filters, filter.NewExcludedFilter(r.name, nil, region.GetStoreIds()))
```

Comment:
`balance region scheduler` use this function, the action should be `balance region`.